api/app/graphql/spree/graphql_config.rb

```diff
@@ -0,0 +1,26 @@
+module Spree
```

Comment:
@gingray let's use Spree Preferences for that  - https://guides.spreecommerce.org/developer/core/preferences.html

Please add those keys to https://github.com/spree/spree/blob/master/api/app/models/spree/api_configuration.rb